### ISSUE TITLE
The Windows client compilation is not working

### ISSUE THREAD

**Issue by tioui** [tioui/Aquaria_Randomizer#4]
The Windows client crash. It look like it has something to do with the JSON received from the server. The server seems to accept de connection. Here is the error on the client command line:

```bash
APClient: Server connected
APClient: Failed to load Õóá─û☻
APClient: [json.exception.parse_error.101] parse error at line 1, column 1: attempting to parse an empty input; check that your input string or stream contains the expected JSON
Segmentation fault
```

**Comment by tioui** [comment]
I have manually compile the randomizer on Windows and it work as intended. But now, it would be necessary to modify the Windows Github workflow to use MSYS2 instead of MinGW on Linux. So I keep this Issue open until the Workflow is fixed.

**Comment by tioui** [comment]
Fixed with [6fcc649](https://github.com/tioui/Aquaria_Randomizer/commit/6fcc6496b09390b570c3753fb902f70483514d1e)
